apps/src/lib/node/ledger/shell/init_chain.rs

```diff
@@ -553,4 +898,218 @@ mod test {
             storage_state.iter(),
         );
     }
+
```

Comment:
can we also test tx signature validation, on bonds and validator account txs?

Reply:
`init_chain` takes finalized genesis files. Those files have already had their signatures checked by the finalization procedure and can be checked by the existing validate genesis command. No signature checks are performed at all during initialization.